### PR TITLE
Aruha 62 expose kafka topics partitions

### PR DESCRIPTION
**Pull request by v-stepanov**
Aligned functionality that exposes partitions information with our latest API; added some tests;
@tmuehl @rcillo @jkliff @CyberDem0n @valgog please take a look

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zalando/nakadi/pull/84" target="_blank">ReviewNinja

File: src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/TopicsControllerAT.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    @Before
+    public void setup() {
+        final KafkaTestHelper kafkaHelper = new KafkaTestHelper(kafkaUrl);
+        actualTopics = kafkaHelper.createConsumer().listTopics();
+    }
+
+    @Test
+    public void whenListTopicsThenOk() throws IOException {
+        // ACT //
+        final Response response = when().get("/topics");
```
Comment (jkliff) [inline src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/TopicsControllerAT.java]: this does not exist anymore on the api

File: src/main/java/de/zalando/aruha/nakadi/controller/EventStreamController.java [hunk 2]
```diff
@@ -86,11 +85,12 @@ public StreamingResponseBody streamEventsFromPartition(
...
+                List<Cursor> cursors = null;
                 if (cursorsStr != null) {
                     try {
-                        cursors = Optional.of(jsonMapper.<List<Cursor>>readValue(cursorsStr,
-                                new TypeReference<ArrayList<Cursor>>() {}));
+                        cursors = jsonMapper.<List<Cursor>>readValue(cursorsStr,
+                                new TypeReference<ArrayList<Cursor>>() {
+                                });
                     } catch (IOException e) {
                         writeProblemResponse(response, outputStream, HttpStatus.BAD_REQUEST.value(),
```
Comment (jkliff) [inline src/main/java/de/zalando/aruha/nakadi/controller/EventStreamController.java]: stream_limit has default of 0
stream_timeout has default of 0 (changed in the api now) 
batch_keep_alive_limit is 0 and was renamed to stream_keep_alive_limit

File: src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/TopicsControllerAT.java (hunk 1 above)
Comment (v-stepanov) [inline src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/TopicsControllerAT.java]: I now removed TopicsController and its test.

File: src/main/java/de/zalando/aruha/nakadi/controller/EventStreamController.java (hunk 2 above)
Comment (v-stepanov) [inline src/main/java/de/zalando/aruha/nakadi/controller/EventStreamController.java]: fixed; aligned parameters to latest API version

File: src/main/java/de/zalando/aruha/nakadi/controller/PartitionsController.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+import static javax.ws.rs.core.Response.Status.SERVICE_UNAVAILABLE;
+import static org.springframework.http.ResponseEntity.ok;
+import static org.zalando.problem.spring.web.advice.Responses.create;
+
+@RestController
+public class PartitionsController {
+
+    private static final Logger LOG = LoggerFactory.getLogger(PartitionsController.class);
+
+    private TopicRepository topicRepository;
```
Comment (CyberDem0n) [inline src/main/java/de/zalando/aruha/nakadi/controller/PartitionsController.java]: Could be `final`

**Pull request closed without merging**